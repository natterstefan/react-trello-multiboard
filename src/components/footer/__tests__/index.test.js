import React from 'react'
import { shallow } from 'enzyme'

import Footer from '../'

describe('Component/MainApp', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
