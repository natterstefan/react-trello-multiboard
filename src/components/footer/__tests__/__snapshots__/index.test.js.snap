// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/MainApp should render without throwing an error 1`] = `
<styled.footer>
  <WithStyles(Typography)
    color="inherit"
  >
    <a
      href="https://github.com/natterstefan/react-trello-multiboard/"
      style={
        Object {
          "color": "#fff",
          "textDecoration": "none",
        }
      }
    >
      <FontAwesomeIcon
        border={false}
        className=""
        fixedWidth={false}
        flip={null}
        icon={
          Array [
            "fab",
            "github",
          ]
        }
        listItem={false}
        mask={null}
        name=""
        pull={null}
        pulse={false}
        rotation={null}
        size="lg"
        spin={false}
        symbol={false}
        transform={null}
      />
       Star on GitHub
    </a>
  </WithStyles(Typography)>
</styled.footer>
`;
